translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"created, if set to 'Never', a regular pod is created. For the latter two --"
+"replicas must be 1.  Default 'Always', for CronJobs `Never`."
+msgstr ""
+"A politica de restart para este Pod.  Possíveis valores [Always, OnFailure, "
+"Never].  Se configurado para 'Always' um deployment é criado, se configurado "
+"para 'OnFailure' um job é criado, se configurado para 'Never', um pod é "
+"criado. Para os dois últimos —replicas deve ser 1.  Valor padrão 'Always', "
+"para CronJobs `Never`."
+
+#: pkg/kubectl/cmd/create_secret.go:88
+msgid "The type of secret to create"
+msgstr "O tipo de segredo para criar"
+
+#: pkg/kubectl/cmd/expose.go:102
+msgid ""
+"Type for this service: ClusterIP, NodePort, or LoadBalancer. Default is "
+"'ClusterIP'."
+msgstr ""
+"Tipo para este serviço: ClusterIP, NodePort, ou LoadBalancer. Valor padrão é "
+"'ClusterIP'."
+
+#: pkg/kubectl/cmd/rollout/rollout_undo.go:72
+msgid "Undo a previous rollout"
+msgstr "Desfazer o rollout anterior"
+
+#: pkg/kubectl/cmd/config/unset.go:48
+msgid "Unsets an individual value in a kubeconfig file"
+msgstr "Remover um valor individual do arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/patch.go:96
+msgid "Update field(s) of a resource using strategic merge patch"
+msgstr "Atualizar o campo(s) de um recurso usando a estratégia de merge patch"
+
+#: pkg/kubectl/cmd/set/set_image.go:95
+msgid "Update image of a pod template"
+msgstr "Atualizar a imagem de um template de pod"
+
+#: pkg/kubectl/cmd/set/set_resources.go:102
+msgid "Update resource requests/limits on objects with pod templates"
+msgstr ""
+"Atualizar os recursos de request/limites em um object com template de pod"
+
+#: pkg/kubectl/cmd/annotate.go:116
+msgid "Update the annotations on a resource"
+msgstr "Atualizar as anotações de um recurso"
+
+#: pkg/kubectl/cmd/label.go:114
+msgid "Update the labels on a resource"
+msgstr "Atualizar os labels de um recurso"
+
+#: pkg/kubectl/cmd/taint.go:87
+msgid "Update the taints on one or more nodes"
+msgstr "Atualizar o taints de um ou mais todos"
+
+#: pkg/kubectl/cmd/create_secret.go:156
+msgid "Username for Docker registry authentication"
+msgstr "Username a autenticação do Docker registry"
+
+#: pkg/kubectl/cmd/apply_view_last_applied.go:64
+msgid "View latest last-applied-configuration annotations of a resource/object"
+msgstr ""
+"Visualizar a última anotação last-applied-configuration de um recurso/objeto"
+
+#: pkg/kubectl/cmd/rollout/rollout_history.go:52
+msgid "View rollout history"
+msgstr "Visualizar o histórico de rollout"
+
+#: pkg/kubectl/cmd/clusterinfo_dump.go:46
+msgid ""
+"Where to output the files.  If empty or '-' uses stdout, otherwise creates a "
+"directory hierarchy in that directory"
+msgstr ""
+"Onde colocar os arquivos de saída. Se vazio ou '-' usa o stdout do terminal, "
+"caso contrário, cria uma hierarquia no diretório configurado"
+
+#: pkg/kubectl/cmd/run_test.go:85
+msgid "dummy restart flag)"
+msgstr "dummy restart flag)"
```

Comment:
missed here? no ideas as this is from the test package

Reply:
untranslated for now